src/components/RegistrationFormBody.tsx

```diff
@@ -1,6 +1,6 @@
-import React from 'react';
-import { Form, Input, Radio, Upload, DatePicker } from 'antd';
+import { DatePicker, Form, Input, Radio, Upload } from 'antd';
```

Comment:
This also looks like it's duplicated, unless I'm mistaken. How is this file used?